dbt_platform_helper/domain/copilot_environment.py

```diff
@@ -134,11 +133,13 @@ class CopilotTemplating:
     def __init__(
         self,
         file_provider: FileProvider = None,
+        io: ClickIOProvider = ClickIOProvider(),
```

Comment:
Would be nice in the future if IOProvider was a Protocol that ClickIOProvider implements. Not for this PR, but to think about.